airflow/contrib/operators/dataproc_operator.py

```diff
@@ -160,10 +160,10 @@ def __init__(self,
                  properties=None,
                  master_machine_type='n1-standard-4',
                  master_disk_type='pd-standard',
-                 master_disk_size=500,
+                 master_disk_size=1024,
```

Comment:
That is really high. The default on the API is I guess 20 GB or 100GB not sure.

Reply:
So I did some checking, for both the master and the worker the default is 500gb. If you open up the Dataproc web UI, this is prefilled. Even if I create a 2TB worker, and a 500gb master, it gives the error. Since 500gb is the default, I'm also okay with keeping it like it was.

Reply:
@DanSedov what's your thoughts?

Reply:
@fenglu-g what are your thoughts? :-)